index.html

```diff
@@ -35,5 +35,7 @@ <h4>Avoid patronizing businesses that support Donald Trump.</h4>
     <div id="map">
       Map.
     </div>
+    <script src="muy_map_conf.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDT5S34GDF7ZpjX-aSMkqvDpNuwjl2kwD0&callback=initMap" async defer></script>

```

Comment:
Any idea on the sensitivity of the API keys here?

Reply:
Sorry, didn't notice you commented.
Not particularly sensitive, this can be used anywhere but only for basic google maps stuff. I don't think that's enough to mine bitcoin on my dime.